src/training/codeReuseOOP/AnimalTree.java

```diff
@@ -0,0 +1,41 @@
+package training.codeReuseOOP;
+
+import java.util.Scanner;
+
+/**
+ * Created by Ivan_Stepantsov on 8/2/2017.
+ */
+public class AnimalTree {
+    public static void main(String[] args) {
+        System.out.println("Lets check animals among next:\n1. Dog\n2. Pigeon\n3. Swordfish\n4. Human");
+        Scanner myScanner = new Scanner(System.in);
+        int n = myScanner.nextInt();
+
+        OrganismOrigin organusmCheck = null;
+        switch (n) {
+            case 1:
+                organusmCheck = new Dog();
+                break;
+            case 2:
+                organusmCheck = new Pigeon();
+                break;
+            case 3:
+                organusmCheck = new Swordfish();
+                break;
+            case 4:
+                organusmCheck = new Human();
+
```

Comment:
again the same. u should check your PR before creation as all the time the same issues